src/main/java/com/dreammaster/gthandler/multiAirFilter/GT_MetaTileEntity_AirFilterBase.java

```diff
@@ -117,104 +157,144 @@ public boolean checkRecipe(ItemStack aStack){
         mEfficiencyIncrease = 10000;
         mEfficiency = 10000 - (getIdealStatus() - getRepairStatus()) * 1000;
         mPollutionReductionWholeCycle = 0;
-        mMaxProgresstime = 200;
-        mEUt=-32;
+        // check pollution for next cycle:
+        hasPollution = getTotalPollution()>=10000;
```

Comment:
What do you think about extracting values like these to a config? Probably would be nice to have other efficiencies and coefficients there too.